ext/DimensionalDataMakie.jl

```diff
@@ -71,10 +71,9 @@ for (f1, f2) in _paired(:plot => :scatter, :scatter, :lines, :scatterlines, :sta
     """
     @eval begin
         @doc $docstring
-        function Makie.$f1(A::AbstractDimVector; axislegendkw=(;), axis = (;), figure = (;), attributes...)
-            args, merged_attributes = _pointbased1(A, attributes)
-            axis_kw, figure_kw = _handle_axis_figure_attrs(merged_attributes, axis, figure)
-            p = Makie.$f2(args...; axis = axis_kw, figure = figure_kw, merged_attributes...)
+        function Makie.$f1(A::AbstractDimVector; axislegendkw=(;), axis=(;), attributes...)
```

Comment:
might also do the same thing for axislegendkw

Reply:
ahh I only got half way with that one